testing/example_scripts/dataclasses/test_compare_dataclasses_with_custom_eq.py

```diff
@@ -10,7 +10,7 @@ class SimpleDataObject:
         field_a: int = field()
         field_b: str = field()
 
-        def __eq__(self, __o: object) -> bool:
+        def __eq__(self, __o: object, /) -> bool:
             return super().__eq__(__o)
```

Comment:
Now that we are using the positinal-only syntax we no longer need the double underscore:

```suggestion
        def __eq__(self, o: object, /) -> bool:
            return super().__eq__(o)
```